packages/kernel-5.15/kernel-5.15.spec

```diff
@@ -85,6 +88,13 @@ Summary: Modules for the Linux kernel
 %description modules
 %{summary}.
 
+%package mkfs-xfs-conf
+Summary: mkfs configurations for the XFS filesystem
+Provides: %{name}(mkfs-xfs-conf)
```

Comment:
Commits don't seem right, this `Provides` is added here and then removed in the last commit of the series.